Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+node {
+  stage('Checkout') {
+    cleanWs()
+    sh 'git lfs install'
+    checkout scm
+  }
+
+  String version = '7.2' // platform version
+  String project = 'project-name' // e.g. "customer-name"
+  String registry = 'harbor.delivery.iqgeo.cloud'
+
+  def shortCommit = sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
+  String buildTag = (new Date()).format('yyyyMMdd').concat("-${shortCommit}")
+
+  stage ('Build: Build Image') {
+    String name = "${registry}/${project}/platform-build"
+    String options = "-f ./deployment/dockerfile.build --pull ."
+
+    buildImage = docker.build(name,  options)
+
+    docker.withRegistry("https://${registry}/", 'harbor-jenkins') {
+      buildImage.push("latest") // not recommended but necessary w/o local caching
+      buildImage.push("${version}")
+      buildImage.push("${version}-${buildTag}")
```

Comment:
This is **_NOT_** the _"right"_ way to do this. This should be done using Tags, but, for fantastically frustrating and still unknown reasons to me, (whether it's the docker plugin version on our [delivery](https://jenkins.delivery.iqgeo.cloud) or something about our configuration/server setup, I can't determine) the [`image.tagsString`](https://www.jenkins.io/doc/pipeline/steps/docker-plugin/#stepclass-dockerbuilderpublisher-build-publish-docker-image) just doesn't seem to do what I'd expect it to.